src/components/RequestGroups/RequestGroupTable.vue

```diff
@@ -229,11 +250,32 @@ export default {
       };
       return defaultQueryParams;
     },
+    updateSemesterOptions: function(data) {
+      for (let semester of data.results) {
+        this.semesterData[semester.id] = semester;
```

Comment:
I think a better way to handle storing semester data + setting the semester options is to update just `this.semesterData` in the `getSemesterOptions()` `.done()` callback and then create a [computed property](https://vuejs.org/v2/guide/computed.html) called `semesterOptions` that generates the semester options based on `this.semesterData`, similar to how `proposalOptions` is generated.